spec/integration_spec.rb

```diff
@@ -109,6 +109,32 @@ def pptx(pptx_name)
     end
   end
 
+  context "with a slide with text" do
+    let(:theme) { OpenXml::Pptx::Parts::Theme.new }
+    let(:master) { OpenXml::Pptx::Parts::SlideMaster.new(theme) }
+    let(:layout) { OpenXml::Pptx::Parts::SlideLayout.new(master) }
+    let(:slide) { OpenXml::Pptx::Parts::Slide.new(layout) }
+
+    before do
+      slide.add_text("Hello World")
```

Comment:
Is `Slide#add_text` kind of a temporary DSL? — seems like it should be more like:

```ruby
slide << TextBox.new.tap do |text_box|
  # ... set position and dimensions of text_box
  text_box << Paragraph.new.tap do |paragraph|
    paragraph << Run.new("Hello World")
  end
end
```

👆 Not based on looking at the real XML, just based on how I imagine it working in my head